DEVELOPMENT.md

```diff
@@ -215,6 +215,36 @@ to start a debugger to which attach from your IDE.
    https://localhost:8081/apis/batch/v1/jobs | jq
    ```
 
+## Enabling Telemetry
+
+1. Install or upgrade KubeArchive adding the following option to the Helm command:
```

Comment:
It looks like this is the first introduction to "zipkin". Could you include a sentence or two here to explain what it is?

```suggestion
## Enabling Telemetry
Zipkin is an open-source unified tracing system. It is used here so each development instance
can gather tracing data without having to set up a centralized data collection system.

1. Install or upgrade KubeArchive adding the following option to the Helm command:
```

Reply:
We are not using Zipkin (the tool), we are using Jaeger. Knative just emits traces using the zipkin protocol, and we use the zipkin compatible endpoint of Jaeger to receive the traces. I can add a generic line about tracing and why we are using tracing if you want.

Reply:
Could you include your explanation in the comment? I think it would be helpful to explain why we are using "zipkin" in the telemetry configuration.

Something like:

"The development instance uses Jaeger to gather telemetry traces. The code uses the zipkin compatible endpoint of Jaeger to receive the traces."

Reply:
Done, check it out.